README.md

```diff
@@ -64,8 +64,12 @@ List of variables used by the role:
 rabbitmq_yum_repo_url: http://www.rabbitmq.com/releases/rabbitmq-server/current/
 
 # Package to install (you can force certain version here)
+# If rabbitmq_yum_repo_url is specified this must be a specific version
 rabbitmq_pkg: rabbitmq-server
 
+# Package to install (you can force certain version here)
+erlang_pkg: erlang
```

Comment:
Shouldn't the `erlang` package pulled automatically be as a dependency of the `rabbitmq-server` package?